_data/meltano/extractors/tap-chargebee/airbyte.yml

```diff
@@ -17,7 +17,7 @@ name: tap-chargebee
 namespace: tap_airbyte
 next_steps: ''
 pip_url: git+https://github.com/MeltanoLabs/tap-airbyte-wrapper.git
-quality: silver
```

Comment:
@pnadolny13 what's contributing to the Airbyte downgrade? I would expect us to keep this at Silver given that Airbyte is actively maintaining these AFAICT.

Reply:
@tayloramurphy I couldnt find the thread where we talked about it but we set minimums of bronze for certain variants like matatika/transferwise/airbyte/singer-io. The tests are in https://github.com/pnadolny13/hub-utils/blob/8ad7f5c543c537ef7653a2243edf38f95a421b09/tests/test_meltano_utils.py#L478. These are getting downgraded because we dont have enough usage data. Its a rolling 3 month window so its possible that people were using it but then stopped.

Reply:
@pnadolny13 makes sense then. Thanks for the reminder.